doc/en/reference.rst

```diff
@@ -889,6 +889,9 @@ Here is a list of builtin configuration options that may be written in a ``pytes
 file, usually located at the root of your repository. All options must be under a ``[pytest]`` section
 (``[tool:pytest]`` for ``setup.cfg`` files).
 
+.. warning:: usage of configuration file different than the canonical, i.e. ``pytest.ini``
```

Comment:
this warning should be specifically for `setup.cfg` tox and pytest use the same ini parser

Reply:
Adding a new commit or amending and force pushing are all valid. :+1: